extensions/algebra/circuit/src/modular_chip/tests.rs

```diff
@@ -73,10 +73,11 @@ fn test_addsub(opcode_offset: usize, modulus: BigUint) {
     let adapter = Rv32VecHeapAdapterChip::<F, 2, 1, 1, BLOCK_SIZE, BLOCK_SIZE>::new(
         tester.execution_bus(),
         tester.program_bus(),
-        tester.memory_controller(),
+        tester.memory_bridge(),
```

Comment:
it's a bit unfortunate we keep needing to pass around `memory_bridge, address_bits`. for later, but should `memory_bridge` just contain` address_bits`?

Reply:
edit: doesn't really make sense for `MemoryBridge` to contain `address_bits` because `MemoryBridge` only cares about `timestamp_bits`. It guess it is just a separate parameter.